acceptance/features/pathway.feature

```diff
@@ -4,13 +4,3 @@ Scenario: Pathway page contains expected text
 Given I am a Cyber Capable Person
 When I visit the Cyber Health Framework pathway page
 Then I see the text "Public Sector Network"
-
-Scenario: Pathway page renders controls
-Given I am a Cyber Capable Person
-When I visit the PSN pathway page
-Then I see a control with id and title "1 Written incident response plan"
-
-Scenario: Pathway page renders subcontrols
-Given I am a Cyber Capable Person
-When I visit the PSN pathway page
-Then I see a SubControl with the title "The organisation has a detailed incident response plan"
```

Comment:
These tests should still pass on this branch as long as the test data has been loaded:
``` shell
python manage.py loaddata */fixtures/testdata*.json
```

I don't think they should be removed as part of this work.

Reply:
That's fair - can we confirm it though?

Reply:
Just committed some changes that add the pathway test back in. Awaiting Travis build outcome

Reply:
tests passing in build [#782](https://travis-ci.com/github/WeAreSnook/cyber-health/builds/227741631)